src/Extension/MultisitesControllerExtension.php

```diff
@@ -54,7 +54,9 @@ public function onAfterInit()
         if (!$this->owner instanceof \SilverStripe\Admin\LeftAndMain) {
             $theme = $site->getSiteTheme();
             if ($theme) {
-                SSViewer::set_themes([$theme, SSViewer::DEFAULT_THEME]);
+                $selectedThemes = explode(',', $theme);
```

Comment:
Any chance that `trim`ing the values will avoid potential issues (in the case that these theme names get used to make paths, etc)?

Reply:
People shouldn't put spaces in the config... but yeah, could be a better catch